### PR TITLE
Add e & n to appinstance key credential objects.

### PR DESCRIPTION
**Pull request by danielzeller-okta**
## Description:
- Add e & n to appinstance key credential objects.
- Related-to: OKTA-77488

## Type of Pull Request:

- [x] Content (Documentation updates or typo-fixes)
- [ ] Blog Post
- [ ] Functional (CSS changes, JS updates, or layout modifications)

### Resolves:
* [OKTA-77488](https://oktainc.atlassian.net/browse/OKTA-77488)

## How Has This Been Tested?
- [ ] I have built this locally and verified that it does not break existing functionality.
- [ ] For functional changes, I have tested against Chrome, Firefox, and Safari, and mobile.
- [ ] For functional changes, I have added tests.

## Primary Reviewers:
@larsjohansen-okta @mystiberry-okta

### CODE REVIEW

**Comment by mystiberry-okta** [comment]
Looks great! Don't forget to switch the base branch from master to weekly (there's an edit button up at the top right of github), and we probably want to add these two items to the property table here: https://github.com/okta/okta.github.io/blob/796fb29a5621b8b9ba36da0291521e4b4171f121/_source/_docs/api/resources/apps.md#application-key-credential-certificate-properties  (descriptions here: https://developer.okta.com/docs/api/resources/oauth2.html#authorization-server-certificate-key-object

File: _source/_docs/api/resources/apps.md [hunk 1]
```diff
@@ -4952,6 +4964,8 @@ The application key credential model defines a [JSON Web Key](https://tools.ietf
 | expiresAt        | timestamp when certificate expires                           | Date                                                                        | FALSE    | FALSE  | TRUE     |           |           |            |
 | x5c              | X.509 certificate chain                                      | Array                                                                       | TRUE     | TRUE   | TRUE     |           |           |            |
 | x5t#S256         | X.509 certificate SHA-256 thumbprint                         | String                                                                      | TRUE     | TRUE   | TRUE     |           |           |            |
+| e                | RSA key value (exponent) for key blinding                    | String                                                                      | FALSE    | FALSE  | TRUE     |           |           |            |
```
Comment (larsjohansen-okta) [inline _source/_docs/api/resources/apps.md]: blinding -> binding

File: _source/_docs/api/resources/apps.md [hunk 2]
```diff
@@ -4952,6 +4964,8 @@ The application key credential model defines a [JSON Web Key](https://tools.ietf
...
+| n                | RSA key value (modulus) for key blinding                     | String                                                                      | FALSE    | FALSE  | TRUE     |           |           |            |
```
Comment (larsjohansen-okta) [inline _source/_docs/api/resources/apps.md]: blinding -> binding

File: _source/_docs/api/resources/apps.md (hunk 1 above)
Comment (larsjohansen-okta) [inline _source/_docs/api/resources/apps.md]: Take it back.

File: _source/_docs/api/resources/apps.md (hunk 2 above)
Comment (larsjohansen-okta) [inline _source/_docs/api/resources/apps.md]: Take it back.

**Comment by cwu-okta** [comment]
LGTM 🍨

**Comment by mystiberry-okta** [comment]
looks great! Squash and merge!

**Pull request merged**